src/node_modules/round-contributors/round-contributors-controller.coffee

```diff
@@ -46,8 +56,18 @@ module.exports = ($scope, $state, config, AuthService, AllocationModel, Allocati
     debug("upload success", response)
     # TODO display success in alerts (see group-members)
     $state.reload('layout')
+    # i want to add an alert after everything is reloaded, have no idea how
+    # $scope.addAlert(
+    #   type: "ya brah",
+    #   msg: "everyone added brah"
+    # )
```

Comment:
also, your `type` in `addAlert` should be one of (primary, success, danger, info, warning, default)

[docs for angular bootstrap alert](https://angular-ui.github.io/bootstrap/#/alert).